src/styles/_variables.scss

```diff
@@ -0,0 +1,23 @@
+$cell-size: 100px;
+$cell-gap: 1px;
+$cell-color: #eee;
+
+$font-family: "Arial", "Helvetica", sans-serif;
+$font-size: 30px;
+
+$graf: $cell-size + $cell-gap;
+
+$calendar-padding: 10px;
+$calendar-margin: 0 auto;
+$calendar-width: 7 * $cell-size + 6 * $cell-gap + 2 * $calendar-padding;
+
+$title-margin: $calendar-margin;
+
+$start-day:
+  "mon" $graf,
```

Comment:
mon should start from 0. Like that:
<img width="1146" alt="Screenshot 2022-07-20 at 12 35 00" src="https://user-images.githubusercontent.com/47606379/179949873-eb50e6af-6bca-40b7-9731-2eac6496edda.png">